src/pages/signup.js

```diff
@@ -163,8 +168,10 @@ const SignUp = () => {
         .catch((error) => {
           // Some error occurred, you can inspect the code: error.code
           // Common errors could be invalid email and invalid or expired OTPs.
-          setForm({...form,
-            message: { text: "There was an error signing up with email link, please try again", type: "error" }})
+          setForm({
```

Comment:
Codacy found an issue: [Unexpected line break after this opening brace.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=6000161)